services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -1309,6 +1309,60 @@ module.exports = {
       return basedb.findOne('platform.Content', { _id: id }, { projection });
     },
 
+    /**
+     *
+     */
+    createContentPromotion: async (_, { input }, { base4rest, basedb }, info) => {
+      validateRest(base4rest);
+      const type = 'platform/content/promotion';
+      const { primarySectionId, companyId, ...payload } = input.payload;
+      const section = await basedb.strictFindById('website.Section', primarySectionId, { projection: { site: 1 } });
```

Comment:
This line can be combine with L1321 in a `Promise.all` so both queries will run in parallel.

Reply:
L1321 queries for the primary site id returned from the section on L1319

Reply:
Nevermind :)